### PR TITLE
Stats: Update visuals for StatsModule

### PR DESCRIPTION
**Pull request by grzegorz-cp**
#### Proposed Changes

* This PR proposes CSS changes to StatsModule until a more advanced solution is developers
* changes are applied only to a `stats-module--unified` parent CSS class and are visible on the main Stats page

#### Testing Instructions

* Visit the instance link below and navigate to `Stats`

Before:
Screenshot 2022-10-26 at 1 46 04 PM

After:
Screenshot 2022-10-26 at 2 18 01 PM

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Related to #

### CODE REVIEW

**Comment by jeffgolenski** [comment]
Screen Shot 2022-10-26 at 10 58 51 AM

This is great @grzegorz-cp, thank you. Before we ship this change, can we make a couple more adjustments?

PR 69459


* Remove the chevron to the right of the title, but keep the title of the card as a link.
* Change the color of the sub-heads below the card titles to `var(—studio-gray-100)` Example: Under Post & Pages the subtitles are "title" and "views"
* Change the card border radius from 4px to 5px

Additional request. I know it's not part of the same component, but visually they're so close. 
* Can you also adjust the border radius on the promotional banner at the footer from 4px to 5px? 


Please ping me or @keoshi for a review after you make these changes! Thanks my friend!

File: client/my-sites/stats/stats-module/style.scss [hunk 1]
```diff
@@ -586,3 +586,81 @@ ul.module-header-actions {
...
+			border-top: 0;
+		}
+
+		&:first-child:last-child {
+			border-radius: 4px;
+		}
+	}
+
+	.card__link-indicator {
+		top: 6px;
```
Comment (jsnmoon) [inline client/my-sites/stats/stats-module/style.scss:638]:
```suggestion
		display: none;
```

For chevron removal as requested by Jeff.

**Review by jsnmoon — COMMENTED** [review]
I noticed this affects some modules present on the Insights page. Could we also apply these changes there? Or limit the changes to just the Traffic page?

image

**Comment by jeffgolenski** [comment]
Changes are looking good! 

Per a discussion with @grzegorz-cp - we're going to ensure these changes also translate to the Insights, Store, and Ads pages before they hit production.

**Comment by grzegorz-cp** [comment]
I extended the styling to all the other pages - Insights, Store and Ads.
Sample images with the changes applied:
Store:
Screenshot 2022-10-27 at 3 55 35 PM

Ads:
Screenshot 2022-10-27 at 3 52 48 PM

Insights:
Screenshot 2022-10-27 at 4 04 52 PM

There are two more boxes on top of the `Insights` page. Do we also want to apply rounded corners and a larger header there?

**Comment by jsnmoon** [comment]
> There are two more boxes on top of the `Insights` page. Do we also want to apply rounded corners and a larger header there?

Yes, let's go ahead and apply the styling changes to the Posting Activity and All-time views cards too.

> Ads:
> [pictured changes]

I'm unsure if this improves the overall layout, especially since the padding around the heading now seems insufficient with the bigger font size. What do you think about leaving the Ads page alone for this PR? We're planning on implementing a more comprehensive layout update for the Ads page in the future, anyway:

image

**Review by jsnmoon — COMMENTED** [review]
Overall code changes look good, but I wonder if there's a way to avoid the stylelint errors without using the `disable-line` comments?

**Comment by grzegorz-cp** [comment]
Thanks for the feedback @jsnmoon. I'm not sure about skipping `Ads` from the update because @jeffgolenski was quite keen on including it. If we skip it, `Ads` would be the only page without rounded corners. On the other hand, you are right about the padding to the headers. I'll increase the padding to match the other headers and let's see how it looks.

Regarding disabling CSS linting checks we would have to add 5px as a new accepted value. Do you think that's a good idea? Moving forward we would probably use it all over Stats to match the designs.

**Comment by jsnmoon** [comment]
> Thanks for the feedback @jsnmoon. I'm not sure about skipping `Ads` from the update because @jeffgolenski was quite keen on including it. If we skip it, `Ads` would be the only page without rounded corners. On the other hand, you are right about the padding to the headers. I'll increase the padding to match the other headers and let's see how it looks.

Ah, good call! Including the changes to the Ads page makes sense in that case. 

> Regarding disabling CSS linting checks we would have to add 5px as a new accepted value. Do you think that's a good idea? Moving forward we would probably use it all over Stats to match the designs.

Is it possible to limit the 5px override to just the stats codebase? It's not a big deal if not, but I'm curious.

**Comment by a8ck3n** [comment]
> Remove the chevron to the right of the title, but keep the title of the card as a link.

One issue with removing the chevron that maybe hasn't been accounted for. When visiting the Insights page with the new styling (chevron hidden), it's not longer possible to tell which headers can be clicked without scrubbing the UI. Unlike the Traffic page, where every header is clickable, only a few should be clickable on the Insights page.

At a glance, can you guess which of the nine modules shown here have a clickable header?

NoChevrons

I'd propose we either leave the chevrons in for now or maybe add some colour to the clickable headers? Any thoughts @jeffgolenski  and/or @keoshi?

**Comment by grzegorz-cp** [comment]
I updated the spacing for headers on the `Ads` page and applied rounded corners and a larger header to the two top (full-width) blocks on the `Insights` page.

Screenshot 2022-10-28 at 5 19 02 PM

**Comment by grzegorz-cp** [comment]
> One issue with removing the chevron that maybe hasn't been accounted for. When visiting the Insights page with the new styling (chevron hidden), it's not longer possible to tell which headers can be clicked without scrubbing the UI. Unlike the Traffic page, where every header is clickable, only a few should be clickable on the Insights page.

That's a very good point. For now, only the cursor pointer informs if the title is clickable. Maybe bold vs normal font-weight could also inform about it?

**Comment by keoshi** [comment]
@a8ck3n Thanks for bringing that up. This is an intermediary state where the titles of the cards are going to be updated but not all cards reflect the final state yet, so it's a tricky transitory phase for sure.

To make sure we're always presenting contextual links, we should add “View all” (and “View all post stats” in the case of latest post”) to the bottom of every card where applicable.

**Comment by jsnmoon** [comment]
@grzegorz-cp: In light of feedback, let's exclude the chevron removal from this PR and consider removing it again when we're swapping out sections with the horizontal bar tables like [here](https://github.com/Automattic/wp-calypso/pull/69545).

**Review by jsnmoon — APPROVED** [review]
@grzegorz-cp: Let's drop the chevron removal from the PR and revisit it when we're replacing individual sections with [the new horizontal bar lists](https://github.com/Automattic/wp-calypso/pull/69545), which should enforce the "View all" links at the bottom of every expandable section that @keoshi pointed out.

Feel free to ship this PR with the chevrons reinstated.

**Comment by jeffgolenski** [comment]
> Let's drop the chevron removal from the PR and revisit it when we're replacing individual sections with https://github.com/Automattic/wp-calypso/pull/69545, which should enforce the "View all" links at the bottom of every expandable section that @keoshi pointed out.

Feel free to ship this PR with the chevrons reinstated.


Good call on that one @jsnmoon, also @a8ck3n thanks for bringing that to our attention

**Pull request merged**